nexus-sdk/src/main/java/io/nexusrpc/handler/ServiceHandler.java

```diff
@@ -172,6 +196,20 @@ public Builder setSerializer(Serializer serializer) {
       return this;
     }
 
+    /**
+     * Add a {@link OperationMiddleware} to the Service Handler. Middlewares are executed per
+     * request in the order they are added.
+     */
+    public Builder addOperationMiddleware(OperationMiddleware middleware) {
+      middlewares.add(middleware);
+      return this;
+    }
+
+    /** Get a list of all {@link OperationMiddleware} registered on this handler. */
+    public List<OperationMiddleware> getMiddlewares() {
+      return middlewares;
+    }
```

Comment:
```suggestion
    /** Get a list of all {@link OperationMiddleware} registered on this handler. */
    public List<OperationMiddleware> getOperationMiddlewares() {
      return operationMiddlewares;
    }
```

I think we should call this "operation middlewares" everywhere like we did on the adder (or can discuss changing the adder name). Would also need to change `ServiceHandler.getMiddlewares` too.

Reply:
done